modules/gltf/register_types.cpp

```diff
@@ -64,22 +64,19 @@ static void _editor_init() {
 
 	// Blend to glTF importer.
 
-	bool blend_enabled = GLOBAL_GET("filesystem/import/blend/enabled");
+	bool blend_enabled = GLOBAL_GET("filesystem/import/blender/enabled");
```

Comment:
I went for `blend` instead of `blender` because that's the file type, like we use `fbx`. If you really want `blender` I'm not against it but there are more files formats we can import from Blender: glTF, and ESCN.